lib/utils/isStandardSyntaxDeclaration.js

```diff
@@ -57,5 +57,10 @@ module.exports = function (decl) {
 		return false;
 	}
 
+	// Sass variables within a namespace (e.g. namespace.$var: x)
+	if (prop.includes('.$') && prop[0] !== '.' && prop[prop.length - 1] !== '$') {
+		return false;
+	}
```

Comment:
I'm not sure what this function should return for invalid declarations such as `.$var` and `namespace.$`.
For now this branch excludes those invalid `prop`s.